server/views/partials/layout.njk

```diff
@@ -41,7 +41,7 @@
 {% block pageTitle %}{{ pageTitle | default(applicationName) }} - {{ applicationName }}{% endblock %}
 
 {% block header %}
-  {% if not feComponents.problemRetrievingData %}
+  {% if not feComponents.problemRetrievingData and feComponents.headerHtml %}
```

Comment:
This fixed a problem where the header and footer were not present on the authorisation error page at all, because they were never looked up (because auth error is trapped quite early, long before the middleware to go to retrieve the header and footer components from the Front End Components service)